docs/workflows/manage.rst

```diff
@@ -3,14 +3,26 @@ Manage Content
 
 Package
 -------
-
+Package (RPM or SRPM) is uniquely identified by NEVRA.
+Within a repository version there can be only one package with the same NEVRA.
+There can't be two packages with same NERVA and different checksum types within a repository version.
 
 Advisory
 --------
 
 
 Modularity Metadata
 -------------------
+RPM plugin will parse a modules.yaml metadata file into modules and modules-defaults.
```

Comment:
Maybe modulemd and modulemd-defaults?
At least this is how they are named in yaml and in pulp3 content types.